src/layouts/instructions.py

```diff
@@ -0,0 +1,53 @@
+"""Layouts for dex instructions data."""
+from construct import Bytes, Int8ul, Int16ul, Int32ul, Int64ul  # type: ignore
+from construct import Struct as cStruct
+from construct import Switch
+
+from .slab import KEY
+
+INITIALIZE_MARKET = cStruct(
+    "baseLotSize" / Int64ul,
```

Comment:
Should these be snake cases? I don't think it's all that important, but I have been using snake cases.

Reply:
I'll change them to make consistent with the slab formats.